modules/nf-core/ltrfinder/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "ltrfinder"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "bioconda::edta=2.2.0"
```

Comment:
Why isn't the module named after edta?

Reply:
The actual tool is LTR_FINDER which is single threaded and can be slow for large genomes. EDTA author has created a multithreaded perl wrapper for LTR_FINDER  and published the wrapper here:  https://github.com/oushujun/LTR_FINDER_parallel. There is no separate Bioconda recipe for LTR_FINDER_parallel.